pkg/cmd/server/server.go

```diff
@@ -223,6 +226,7 @@ func NewCommand(f client.Factory) *cobra.Command {
 	command.Flags().DurationVar(&config.garbageCollectionFrequency, "garbage-collection-frequency", config.garbageCollectionFrequency, "How often garbage collection is run for expired backups.")
 	command.Flags().BoolVar(&config.defaultVolumesToFsBackup, "default-volumes-to-fs-backup", config.defaultVolumesToFsBackup, "Backup all volumes with pod volume file system backup by default.")
 	command.Flags().StringVar(&config.uploaderType, "uploader-type", config.uploaderType, "Type of uploader to handle the transfer of data of pod volumes")
+	command.Flags().DurationVar(&config.resourceTimeout, "resource-timeout", config.resourceTimeout, "How long to wait for certain resource processes before timeout.")
```

Comment:
Please clarify in the help text what the default value will be if this is not set, and also the relationship with other "timeout" param like `defaultCSISnapshotTimeout `, `resourceTerminatingTimeout`, `podVolumeOperationTimeout `

Reply:
The default value should be listed here in the help text, I'd agree. I'm not 100% sure what we'd want to say about the relationship with other parameters, but maybe something like this:
"The default value is 5 minutes (or 10?). This parameter covers several cases involving resource creation or readiness that are not covered by the other more specific timeout parameters."

Reply:
+1 on stating the default value in `velero server help`

Reply:
updated